openstack/assets/dashboards/openstack-controller.json

```diff
@@ -0,0 +1,693 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "OpenStack Controller Overview",
+    "created": "2020-04-22T15:12:53.212166+00:00",
+    "created_by": {
+        "access_role": "st",
+        "disabled": false,
+        "email": "support@datadoghq.com",
+        "handle": "support@datadoghq.com",
+        "is_admin": false,
+        "name": "Datadog",
+        "role": null,
+        "title": null,
+        "verified": true
+    },
+    "description": "## OpenStack Controller - Overview\n\nPreset dashboard for the OpenStack Controller integration. Used for OpenStack deployments v13 and higher. \n\n[See integration docs for more details](https://docs.datadoghq.com/integrations/openstack_controller/)",
+    "id": 1065652,
+    "modified": "2020-04-22T15:19:50.067638+00:00",
+    "new_id": "ekn-bqw-h7v",
+    "read_only": false,
+    "template_variables": [
+        {
+            "default": "*",
+            "name": "project",
+            "prefix": "project_name"
+        },
+        {
+            "default": "*",
+            "name": "aggregate",
+            "prefix": "aggregate"
+        }
+    ],
+    "widgets": [
+        {
+            "bgcolor": "gray",
+            "font_size": "36",
+            "height": 8,
+            "html": "\n[Hypervisors](https://www.datadoghq.com/blog/openstack-monitoring-nova/#hypervisor-metrics)",
+            "id": 0,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 94,
+            "x": 21,
+            "y": 2
+        },
+        {
+            "height": 14,
+            "id": 1,
+            "tile_def": {
+                "requests": [
+                    {
+                        "conditional_formats": [
+                            {
+                                "comparator": "<",
```

Comment:
The metric is the amount of used memory, why showing it in red when the value is low?

Reply:
@FlorianVeaux this is identical to the way this widget is displayed for the default OpenStack dashboard currently in production.